zbus/Cargo.toml

```diff
@@ -26,6 +26,8 @@ camino = ["zvariant/camino"]
 bus-impl = ["p2p"]
 # Enables API that is only needed for peer-to-peer (p2p) connections.
 p2p = []
+# Enables API that is only needed for IBus Daemon.
+ibus = []
```

Comment:
Nice! Please make sure the commit to introduce this feature comes before use of it, or alternatively you can just squash this into that commit. Just make sure to mention in the commit message details that the new function is  only enabled by the new cargo feature, `ibus`.